src/msg/async/ProtocolV1.cc

```diff
@@ -957,6 +957,12 @@ CtPtr ProtocolV1::handle_message_footer(char *buffer, int r) {
   message->set_throttle_stamp(throttle_stamp);
   message->set_recv_complete_stamp(ceph_clock_now());
 
+  connection->logger->tinc(l_msgr_all_read_lat,
+                           ceph_clock_now() - recv_stamp);
+  log_latency(__func__,
+    ceph_clock_now() - recv_stamp,
```

Comment:
There is a possibility for having a discrepancy with the value we use to increase `l_msgr_all_read_lat`. It should be calculated once.

Reply:
done